replica/table.cc

```diff
@@ -592,6 +592,8 @@ table::seal_active_memtable(flush_permit&& flush_permit) noexcept {
         try_flush,
         done
     };
+    // Try flushing for up to 5 minutes
+    auto timeout = db::timeout_clock::now() + 300s;
     for (auto st = state::add_memtable; st != state::done;) {
```

Comment:
5 minutes is a way too little, imagine a huge memtable filled with tombstones and/or a very slow disk.

abort on timeout should be extremely rare.

Reply:
Maybe it's better to count the number of retries instead.

Reply:
Can do

Reply:
Done in v9